src/sentry/tasks/derive_code_mappings.py

```diff
@@ -146,15 +146,21 @@ def set_project_codemappings(
             },
         )
 
-        RepositoryProjectPathConfig.objects.create(
```

Comment:
This is what would cause the [IntegrityErrors](https://sentry.io/organizations/sentry/issues/3727959102/?project=1&query=is%3Aunresolved+derive+level%3Aerror&referrer=issue-stream&statsPeriod=5h).